mmv1/third_party/terraform/tests/resource_spanner_database_test.go.erb

```diff
@@ -91,7 +124,14 @@ resource "google_spanner_database" "basic" {
   ]
   deletion_protection = false
 }
-`, instanceName, instanceName, databaseName)
+
+resource "google_spanner_database" "basic_spangres" {
```

Comment:
Rather than configuring a new resource within an existing config and within an existing test case, can you split this out to a new configuration and test case? That will run them in parallel rather than run them serially (and require complicated cleanup)